src/ownership-and-deconstruction.rst

```diff
@@ -637,6 +637,14 @@ An :dt:`extending expression` is either
   The :t:`tail expression` of a :t:`block expression` that is an
   :t:`extending expression`.
 
+* :dp:`fls_eeaJtK4w5gVK`
+  The :t:`argument operand` to a :t:`tuple struct call expression`
+  of an extending :t:`tuple struct`.
+
+* :dp:`fls_rT33l0YwUGlK`
+  The :t:`argument operand` to a  :t:`tuple struct call expression`
+  of an extending :t:`tuple enum variant`.
+
```

Comment:
This wording makes it sound like the tuple struct itself can be extending or not extending, but it really is the constructor expression and not the tuple struct itself that is extending.  In the cases above that use this "a X of Y" phrasing, "Y" is still an expression, but that's not the case here.

Reply:
I didn't see a glossary entry for a "tuple enum variant call expression", so maybe that would need to added so it can say an "extending tuple struct call expression" and an "extending tuple enum variant call expression".

Or, alternatively, I suppose, it could say, e.g., "an extending call expression except for one that invokes a function other than the constructor for a tuple struct or tuple enum variant", but that's maybe a bit roundabout.

Reply:
1d6b3dce7cd4f3593906ad3821a8d21a65c8b60a